File: problem_seventeen.js
// count vawels and onsonats  
var inp ="hello payer bhai"
function myFunction(inp){
    var value =inp.split(/\s/).join('').split("")
    var count = ""
    var count1 = ""
    for(var i = 0; i < value.length;i++){
        if(value[i ]=="a" || value[i]=="e" || value[i]=="i" || value[i]=="o" || value[i]=="u" ){
            count+=value[i]
        }else{
            count1 +=value[i]
        }
       }
    console.log("Vowels ->",count)
    console.log("Cosoants ->",count1)
}
myFunction(inp)

File: problem_seventeen.py
# count vowels and consonants
inp = input("Enter a sentence:")
def MyFunction(inp):
    val = list(inp.replace(" ", ""))
    count = ""
    count1 =""
    for i in val:
        if i == "a" or i == "e" or i =="i" or i =="o" or i =="u":
            count += i
        else :
            count1+=i    
    print("Vowels -> ",count)
    print("Consonants ->",count1)
MyFunction(inp) 